ci-operator/config/openshift/secrets-store-csi-driver/openshift-secrets-store-csi-driver-main.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.20-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+binary_build_commands: make build
+build_root:
+  from_repository: true
```

Comment:
Why not add the line `canonical_go_repository: sigs.k8s.io/secrets-store-csi-driver` after this line? (see this place in a similar file https://github.com/openshift/release/blob/master/ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml#L13)

Reply:
Thanks, I added it in a new commit. I actually wasn't certain about this line, as we have a few drivers like google filestore without it, but it appears to be [purely cosmetic](https://github.com/openshift/ci-tools/blob/a8bae5fb2eaec077982e90430dde7407d5d82fd2/pkg/api/types.go#L58-L62).